pkg/cloudprovider/aws.go

```diff
@@ -142,31 +175,37 @@ func (p *AWSProvider) Test(secret *kapi.Secret) error {
 	case BackupStorage:
 		key, _ := uuid.NewUUID()
 		test := S3Test{
-			key:    key.String(),
-			url:    p.GetURL(),
-			region: p.Region,
-			bucket: p.Bucket,
-			secret: secret,
+			key:            key.String(),
+			url:            p.GetURL(),
+			region:         p.GetRegion(),
+			disableSSL:     p.GetDisableSSL(),
+			forcePathStyle: p.GetForcePathStyle(),
+			bucket:         p.Bucket,
+			secret:         secret,
 		}
 		err = test.Run()
 	case VolumeSnapshot:
-		test := Ec2Test{
+		// Disable volume snapshot test until
```

Comment:
Removing VSL verification since it's never being used in the migration process and is broken until we make UX changes.